rivet.spec

```diff
@@ -11,44 +11,26 @@ Patch0: rivet-1.4.0
 %endif
 
 %if "%{?cms_cxxflags:set}" != "set"
-%define cms_cxxflags -O2 -std=c++11
+%define cms_cxxflags -O2 -std=c++0x
```

Comment:
ouch, some of the changes here and below happened in a way I don't understand as I did not did them explicitly ... gosh